papas/display/src/GDetectorElement.cpp

```diff
@@ -33,8 +33,8 @@ GDetectorElement::GDetectorElement(std::shared_ptr<const DetectorElement> detEle
     double radius = elem.radius();
     double dz = elem.z();
 
-    m_circles.push_back(std::move(TEllipse(0., 0., radius, radius)));
-    m_boxes.push_back(std::move(TBox(-dz, -radius, dz, radius)));
+    m_circles.push_back(TEllipse(0., 0., radius, radius));
+    m_boxes.push_back(TBox(-dz, -radius, dz, radius));
```

Comment:
Same. Note that I don't think there was a double construction, but in any case this is better.